index.js

```diff
@@ -29,4 +31,38 @@ app.use(routes);
 sequelize.sync({ force: false }).then(() => {
   // We need our http server to listen since express isn't creating it's own.
   httpServer.listen(PORT, () => console.log(`Listening on port ${PORT}`));
-});
\ No newline at end of file
+});
+
+const deleteUsersWithEmailContainingStringOlderThanTwoHours = async function(str) {
+  const twoHoursAgo = new Date(Date.now() - 2 * 60 * 60 * 1000);
+
+  const usersToDelete = await User.findAll({
+    where: {
+      email: {
+        [Op.like]: `%${str}%`
+      },
+      createdAt: {
+        [Op.lt]: twoHoursAgo
+      }
+    }
+  });
+  console.log(usersToDelete)
+
+  await Promise.all(usersToDelete.map(user => user.destroy()));
+}
+
+
+// Define the function to run every 1 hour
+const deleteUsersTask = () => {
+  deleteUsersWithEmailContainingStringOlderThanTwoHours('@gridlocke.net')
```

Comment:
While we will probably never end up offering emails, technically a gridlocke.net email could potentially be a valid email since we own the domain, maybe consider something a little more unique that would never be used, like "@gridlocke.guest"... or even "@guest" if we only have email validation on the front end.